coordinator/src/position/models.rs

```diff
@@ -61,6 +60,7 @@ pub struct Position {
     pub expiry_timestamp: OffsetDateTime,
     pub update_timestamp: OffsetDateTime,
     pub trader: PublicKey,
+    pub coordinator_leverage: f32,
```

Comment:
We should probably reorganise and re-document this struct to reflect the fact that it is no longer just the trader's position.